docassemble/AffidavitOfIndigencySupplement/data/questions/affidavit-of-indigency-supplement.yml

```diff
@@ -612,22 +612,22 @@ code: |
 ---
 id: expenses control
 question: |
-  What expenses did you pay in the last 12 months?
+  What are your monthly expenses?
 fields:
-  - Rent, Mortgage, and Utilities: users[0].has_fixed_expenses
+  - Rent, Mortgage, Utilities, and Telephone: users[0].has_fixed_expenses
     datatype: yesnowide
-  - Health insurance and Uninsured Medical Expenses,: users[0].has_health_expenses
+  - Health insurance and Uninsured Medical Expenses: users[0].has_health_expenses
     datatype: yesnowide
-  - Food, Clothing, Laundry, Cleaning, and Telephone: users[0].has_variable_expenses
+  - Food, Clothing, Laundry, and Cleaning: users[0].has_variable_expenses
     datatype: yesnowide
   - Transportation and Car Insurance: users[0].has_transport_expenses
     datatype: yesnowide
   - Child Education, Child Support, and Child Care: users[0].has_child_expenses
     datatype: yesnowide
 ---
 id: fixed expenses amount
-question: "Fixed expenses"
-subquestion: "How much did you pay in the last 12 months?"
+question: "Rent, Mortgage, Utilities, and Telephone"
+subquestion: "What are your expenses?"
```

Comment:
Is this meant to be "monthly expenses"? Or is that just for that first question? I think it may be good to repeat it on each page.